### PR TITLE
Windows support

### PR DESCRIPTION
**Pull request by vcernomschi**
Update initHook to support Windows

### CODE REVIEW

File: src/DeepNgRoot/hook.init.js [hunk 1]
```diff
@@ -24,6 +27,8 @@ var exports = module.exports = function(callback) {
         return;
       }
 
+      console.log('No errors, will run "npm root -g"')
```
Comment (devs-deep) [inline src/DeepNgRoot/hook.init.js]: Codacy Issue found: [Missing semicolon.](https://www.codacy.com/app/MitocGroup/deep-microservices-root-angularjs/file/1796871863/issues/source?bid=2928423&fileBranchId=3072037#l30)

File: src/DeepNgRoot/hook.init.js [hunk 2]
```diff
@@ -1,22 +1,25 @@
 /**
  * Created by AlexanderC on 10/6/15.
  */
- 
+
 'use strict';
 
 var exports = module.exports = function(callback) {
```
Comment (devs-deep) [inline src/DeepNgRoot/hook.init.js]: Codacy Issue found: ["exports" is defined but never used](https://www.codacy.com/app/MitocGroup/deep-microservices-root-angularjs/file/1796871863/issues/source?bid=2928423&fileBranchId=3072037#l7)

File: src/DeepNgRoot/hook.init.js [hunk 3]
```diff
@@ -1,23 +1,33 @@
...
   console.log('Checking for ' + DEEP + ' globally');
 
-  var installation = exec(
-    'npm list -g --depth 1 ' + DEEP + ' > /dev/null 2>&1 || npm install -g ' + DEEP + ' --production --loglevel warn &>/dev/null',
-    function(error) {
+  var cmd = 'npm list -g --depth 1 ' + DEEP + ' > /dev/null 2>&1' +
+    '|| npm install -g ' + DEEP + ' --production --loglevel warn &>/dev/null';
+
+  //remove redirection stdout and stderr for Windows
+  if (os.platform().indexOf('win32') > -1 ||
```
Comment (AlexanderC) [inline src/DeepNgRoot/hook.init.js]: Please use `!== -1`

File: src/DeepNgRoot/hook.init.js [hunk 4]
```diff
@@ -1,23 +1,33 @@
...
+    os.platform().indexOf('win64') > -1) {
+
+    cmd = cmd.replace(/\s*>\s*\/dev\/null\s*2>\&1/gi, ' ');
```
Comment (AlexanderC) [inline src/DeepNgRoot/hook.init.js]: This may lead to String Buffer Overflow exception on more output

**Pull request merged**